tests/acceptance/features/coreApiShareManagementToShares/moveReceivedShare.feature

```diff
@@ -19,9 +19,8 @@ Feature: sharing
     And user "Brian" accepts share "/TMP" offered by user "Alice" using the sharing API
     And user "Carol" accepts share "/TMP" offered by user "Alice" using the sharing API
     And user "Brian" moves folder "/Shares/TMP" to "/Shares/new" using the WebDAV API
-    And the administrator deletes user "Carol" using the provisioning API
-    Then the OCS status code of responses on all endpoints should be "100"
-    And the HTTP status code of responses on all endpoints should be "200"
+    And the administrator deletes user "Carol" using the Graph API
+    And the HTTP status code of responses on all endpoints should be "204"
```

Comment:
Aren't OCS status code for user deletion 0 since that endpoint is not being used for the task?

Reply:
Yes the deletion will use the dav path so it won't return the ocs code but that should not be the reason why we don't check the status code of the sharing step because they should send `100`. Removing it is not the solution check `ocs` status code for the sharing endpoints and HTTP for the rest?
```
Then the OCS status code of responses on all endpoints should be "100"
```